src/components/Body.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import { InstantSearch } from 'react-instantsearch-dom'
+
+import { useMeilisearchClientContext } from 'context/MeilisearchClientContext'
+import Box from 'components/Box'
+import Header from 'components/Header/index'
+import BodyWrapper from 'components/BodyWrapper'
+import EmptyView from 'components/EmptyView'
+import OnBoarding from 'components/OnBoarding'
+import Results from 'components/Results'
+import Typography from 'components/Typography'
+
+const IndexContent = ({ currentIndex }) => {
+  if (!currentIndex) return <OnBoarding />
+  if (currentIndex?.stats?.numberOfDocuments > 0) return <Results />
+  return (
+    <EmptyView buttonLink="https://docs.meilisearch.com/reference/api/documents.html">
+      <Typography
+        variant="typo8"
+        style={{ textAlign: 'center' }}
+        mb={32}
+        color="gray.0"
+      >
+        There’s no document in the selected index
+      </Typography>
+    </EmptyView>
+  )
+}
+
+const Body = ({
+  currentIndex,
+  indexes,
+  getIndexesList,
+  setCurrentIndex,
+  requireApiKeyToWork,
+  isApiKeyBannerVisible,
+}) => {
+  const { meilisearchJsClient, instantMeilisearchClient } =
+    useMeilisearchClientContext()
+
+  return (
+    <InstantSearch
+      indexName={currentIndex ? currentIndex.uid : ''}
+      searchClient={instantMeilisearchClient}
+    >
+      <Header
+        indexes={indexes}
+        currentIndex={currentIndex}
+        setCurrentIndex={setCurrentIndex}
+        requireApiKeyToWork={requireApiKeyToWork}
+        client={meilisearchJsClient}
+        refreshIndexes={getIndexesList}
+        isBannerVisible={isApiKeyBannerVisible}
+      />
+      <BodyWrapper>
+        {/* <Sidebar /> */}
```

Comment:
Sidebar?

Reply:
It's legacy code, we wanted to add a sidebar with facets and then decided not to. I chose to comment it in case we want to enable it again in the future. 
So no change here with this PR :)